packages/vue-compat/__tests__/global.spec.ts

```diff
@@ -500,3 +500,11 @@ test('local app config should not affect other local apps in v3 mode', () => {
   const app2 = createApp({})
   expect(app2.config.globalProperties.test).toBe(undefined)
 })
+
+test('ATTR_ENUMERATED_COERCION: true', () => {
```

Comment:
I'm wondering whether the tests for this should be in `misc.spec.ts` instead, as there are already tests for `ATTR_ENUMERATED_COERCION` in that file?